locksmith/src/utils/openSea.ts

```diff
@@ -0,0 +1,21 @@
+import networks from '@unlock-protocol/networks'
+
+export const generateOpenSeaUrl = ({
+  lockAddress,
+  tokenId,
+  network,
+}: {
+  lockAddress: string
+  tokenId: string
+  network: any
+}): string | undefined => {
+  const isTestNetwork = [4, 31337, 80001].includes(network)
+
+  const baseUrl = isTestNetwork
```

Comment:
Can you check the network package and put that logic there please do it's reusable?

Thank you!

Reply:
added!